wls-gui-wahllokalsystem/tests/components/wahlvorbereitung/BaseWahleroeffnungCard.spec.ts

```diff
@@ -157,5 +171,106 @@ describe("BaseWahleroeffnungCard.vue", () => {
 
       expect(wahlbezirkStore.sendEroeffnungsuhrzeit).toHaveBeenCalled();
     });
+
+    it("should_setIsZuSpaet_when_eroeffnungsuhrzeitIsToLate", async () => {
+      const infomanagementStore = useInfomanagementStore();
+      // @ts-expect-error: cannot set readonly
+      infomanagementStore.fruehesteEroeffnungsuhrzeit = "07:00:00";
+      // @ts-expect-error: cannot set readonly
+      infomanagementStore.fruehesteSchliessungsuhrzeit = "08:00:00";
+
+      const wahlbezirkStore = useWahlbezirkStore();
+      wahlbezirkStore.eroeffnungsuhrzeit = new Date();
+      wahlbezirkStore.eroeffnungsuhrzeit.setHours(7);
+      wahlbezirkStore.eroeffnungsuhrzeit.setMinutes(30);
+      wahlbezirkStore.eroeffnungsuhrzeit.setSeconds(0);
```

Comment:
ich glaube du kannst das auch einfach so schreiben
```suggestion
      wahlbezirkStore.eroeffnungsuhrzeit.setHours(7, 30, 0);
```

Reply:
[f5fab64](https://github.com/it-at-m/Wahllokalsystem/pull/1567/commits/f5fab649e30fab06ab8fe938d4a0cabddf2c3753)